packages/cloud-cognitive/src/components/Datagrid/styles/addons/_RowSizeDropdown.scss

```
@@ -16,7 +16,7 @@
   box-shadow: 1px 4px 8px -3px $overlay-01, -1px 6px 8px -5px $overlay-01;
 }
 
-.#{$block-class}__row-size-button--open {
+.#{$carbon-class}--btn--ghost.#{$block-class}__row-size-button--open {
```

This should still work if you just use 

```suggestion
.#{$carbon-prefix}--btn--ghost.#{$block-class}__row-size-button--open {
```

which is how we usually have it in other parts of the Datagrid component.

Then you can remove `$carbon-class` from variables since I don’t think we need to add this one separately?